src/SSRv1/templates/bulk/import.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/** @var \WEEEOpen\Tarallo\User $user */
+/** @var array|null $imports */
+$imports = $imports ?? null;
+$this->layout('main', ['title' => 'Import from Peracotta', 'user' => $user, 'currentPage' => 'bulk import']);
+?>
+<div class="row">
+	<div class="col">
+		<table class="table table-borderless stats">
+			<caption>Bulk imports from PERACOTTA</caption>
```

Comment:
Remove "from PERACOTTA", the master plan is to use this API for other kinds of imports too :smirk: